src/about/coc.md

```diff
@@ -1,4 +1,4 @@
-# Code Of Conduct {#code-of-conduct}
+# Code Of Conduct {#code-of-conduct} 
```

Comment:
**question**: Could you please provide more context on why we need whitespace here?